inc/Engine/Plugin/UpdaterSubscriber.php

```diff
@@ -520,9 +520,9 @@ public function rollback_add_return_link( $update_actions ) {
 	 *
 	 * @return mixed|WP_Error
 	 */
-	public function upgrade_pre_install_option( $return, $plugin ) {
+	public function upgrade_pre_install_option( $return, $plugin = null ) {
```

Comment:
since the type of the second argument should be an array, the default value for the parameter should be an empty array instead of a null value